groovy/postInit/components/OpAmpCircuits.groovy

```diff
@@ -221,24 +230,41 @@ mods.gregtech.circuit_assembler.removeByInput(24, [metaitem('circuit_board.good'
 mods.gregtech.circuit_assembler.removeByInput(24, [metaitem('circuit_board.good'), metaitem('circuit.basic_integrated') * 2, metaitem('component.resistor') * 2, metaitem('component.diode') * 2, metaitem('wireFineGold') * 4, metaitem('boltSilver') * 4], [fluid('tin') * 144])
 
 CIRCUIT_ASSEMBLER.recipeBuilder()
-        .inputs(metaitem('circuit.basic_integrated') * 2, metaitem('component.resistor') * 2, metaitem('component.diode') * 2, metaitem('wireFineAnnealedCopper') * 4)
+        .inputs(metaitem('circuit.basic_integrated') * 2, ore('component.resistor') * 2, ore('component.diode') * 2, ore('wireFineAnnealedCopper') * 4)
         .inputs(metaitem('voltage_regulator.mv'))
         .fluidInputs(fluid('soldering_alloy') * 72)
         .outputs(metaitem('circuit.good_integrated') * 2)
-        .duration(400)
+        .duration(100)
+        .EUt(24)
+        .buildAndRegister()
+
+CIRCUIT_ASSEMBLER.recipeBuilder()
+        .inputs(metaitem('circuit.basic_integrated') * 2, ore('component.resistor') * 2, ore('component.diode') * 2, ore('wireFineAnnealedCopper') * 4)
+        .inputs(metaitem('voltage_regulator.mv'))
+        .fluidInputs(fluid('tin') * 144)
+        .outputs(metaitem('circuit.good_integrated') * 2)
+        .duration(100)
         .EUt(24)
         .buildAndRegister()
 
 // Advanced Integrated Circuit * 1
 mods.gregtech.circuit_assembler.removeByInput(30, [metaitem('circuit.good_integrated') * 2, metaitem('plate.integrated_logic_circuit') * 2, metaitem('plate.random_access_memory') * 2, metaitem('component.transistor') * 4, metaitem('wireFineElectrum') * 8, metaitem('boltAnnealedCopper') * 8], [fluid('soldering_alloy') * 72])
 mods.gregtech.circuit_assembler.removeByInput(30, [metaitem('circuit.good_integrated') * 2, metaitem('plate.integrated_logic_circuit') * 2, metaitem('plate.random_access_memory') * 2, metaitem('component.transistor') * 4, metaitem('wireFineElectrum') * 8, metaitem('boltAnnealedCopper') * 8], [fluid('tin') * 144])
 
-
 CIRCUIT_ASSEMBLER.recipeBuilder()
-        .inputs(metaitem('circuit.good_integrated') * 2, metaitem('component.inductor') * 2, metaitem('component.diode') * 2, metaitem('boltElectrum') * 8)
+        .inputs(metaitem('circuit.good_integrated') * 2, ore('component.inductor') * 2, ore('component.diode') * 2, ore('boltElectrum') * 8)
         .inputs(metaitem('voltage_regulator.hv'))
         .fluidInputs(fluid('soldering_alloy') * 72)
         .outputs(metaitem('circuit.advanced_integrated') * 2)
-        .duration(800)
-        .EUt(30)
+        .duration(200)
+        .EUt(120)
+        .buildAndRegister()
+
+CIRCUIT_ASSEMBLER.recipeBuilder()
+        .inputs(metaitem('circuit.good_integrated') * 2, ore('component.inductor') * 2, ore('component.diode') * 2, ore('boltElectrum') * 8)
```

Comment:
component.inductor should be componentInductor

Reply:
![image](https://github.com/user-attachments/assets/065b0d91-b4af-4238-b096-ab306a28f474)
it should be component.inductor tho?